WebViewControl/WebView.cs

```diff
@@ -72,6 +72,7 @@ public partial class WebView : IDisposable {
         public event Action TitleChanged;
         public event UnhandledAsyncExceptionEventHandler UnhandledAsyncException;
         public event Action</*url*/string> PopupOpening;
+        public event ConsoleMessageEventHandler OnConsoleMessage;
```

Comment:
Missing the `ConsoleMessageEventHandler`

Also `OnConsoleMessage` should be `ConsoleMessageEmitted`

Reply:
Hello, I changed OnConsoleMessage to ConsoleMessageEmitted. 

For me Xilium.CefGlue.Common.Events has ConsoleMessageEventHandler and there is an import to it in WebView.cs

_using Xilium.CefGlue.Common.Events;_

Would it be recommended to recreate it in the webview project? Thank you.

Reply:
Hi,

Yes. The webview doesn't/shouldn't expose anything from CefGlue

Reply:
Thank you. I think I got everything properly recreated in the webview control in latest commit.